packages/eddsa-pcd/src/EdDSAPCD.ts

```diff
@@ -161,3 +163,10 @@ export const EdDSAPCDPackage: PCDPackage<
   deserialize,
   init
 };
+
+export function getEdDSAPublicKey(privateKey: string): [string, string] {
```

Comment:
Possibly something I should have done on my PR, but perhaps the `[string, string]` type for the signature could be defined as a named type.

Reply:
i can do that!